jest.config.js

```diff
@@ -0,0 +1 @@
+module.exports = require('@folio/stripes-acq-components/jest.config');
```

Comment:
I don't think the idea of reusing the config from stripes-acq-components is good. As ui-inventory may end up with its own mocks and setup and in order to change them the changes would need to occur in that module which does not necessarily needed for stripes-acq-components. The first thing which I already found which differs between two modules is that babel-config should use legacy decorators as it is in many places the `@stripesConnect` notation is used.

This is how I did it in data-export.
```
  plugins: [
    [
      '@babel/plugin-proposal-decorators',
      { legacy: true },
    ],
    '@babel/plugin-proposal-class-properties',
    '@babel/plugin-transform-runtime',
  ],
```

Reply:
I'm not sure that I've got your point. I've added initial setup for tests, any dev can change, fix or improve it. It works for me, and I'm not familiar with things you described, so I prefer to pass the buck of changing this. 
However, I can revert the PR if you think it's the best way to handle

Reply:
No need to revert. I think you did well to set up it in a way it fulfills your need. I just think this can be improved to fulfill the needs of testing other more complex components. I think it deserves its own story.

Reply:
sure, np. own story is nice